ipn/localapi/localapi.go

```diff
@@ -73,6 +73,7 @@ var handler = map[string]localAPIHandler{
 	"dev-set-state-store":         (*Handler).serveDevSetStateStore,
 	"dial":                        (*Handler).serveDial,
 	"file-targets":                (*Handler).serveFileTargets,
+	"flush-logtail":               (*Handler).serveFlushLogtail,
```

Comment:
Do we need this LocalAPI method? As far as I can tell it's unused. Or were you thinking of add a `tailscale debug` command to also allow log flushing?

Reply:
Yeah, this is mostly for debugging. But I'm also going to make "tailscale bugreport" etc do it.

I suppose I could remove this one and instead make this PR do it on bugreport.

Reply:
Done